pyoozie/tags.py

```diff
@@ -3,16 +3,31 @@
 from __future__ import unicode_literals
 from abc import ABCMeta, abstractmethod
 import re
+import string  # pylint: disable=deprecated-module
```

Comment:
Some functions are deprecated in [Python 2](https://docs.python.org/2/library/string.html#deprecated-string-functions) and [Python 3](https://docs.python.org/3/library/string.html) but the module isn't deprecated, [Pylint is just a bit heavy-handed](https://www.logilab.org/ticket/2481).